Intersect (Core)/Network/Packets/Server/MapLayerChangedPacket.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MessagePack;
+using System;
+using System.Collections.Generic;
+
+namespace Intersect.Network.Packets.Server
+{
+    [MessagePackObject]
+    public class MapLayerChangedPacket : IntersectPacket
```

Comment:
This is still called MapLayer after your other changes regarding naming conventions.

Reply:
Done